runtime/doc/eval.txt

```diff
@@ -1552,6 +1552,9 @@ v:event		Dictionary of event data for the current |autocommand|.  Valid
 					operation.
 			regtype		Type of register as returned by
 					|getregtype()|.
+			completeindex   Index of current selected complete
+					item, Is `-1` when no complete item
+					selected.
```

Comment:
It's good to use `v:event`, but `v:completed_item` already exists and holds more info, so maybe it should be used instead.

Also, `v:event` items can be more generic, so it could be named or `index`. Because `v:event` is context-dependent. Then the help would be like this:

```
index    Event-specific current index.
```

Reply:
Changed to use `completeitem` in `v:event`.